### PR TITLE
Refactor asg docs kops

### PR DESCRIPTION
**Pull request by andreipope**

### CODE REVIEW

File: content/portworx-install-with-kubernetes/cloud/aws/aws-kops/plan.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+{{% content "cloud-references/auto-disk-provisioning/shared/automatic-disk-provisioning-intro.md" %}}
+
+### Why would I need this?
+
+{{% content "cloud-references/auto-disk-provisioning/shared/automatic-disk-provisioning-why-need.md" %}}
+
+### How do I get started?
+
+{{% content "cloud-references/auto-disk-provisioning/shared/automatic-disk-provisioning-set-up-intro.md" %}}
```
Comment (harsh-px) [inline content/portworx-install-with-kubernetes/cloud/aws/aws-kops/plan.md]: this section shows generating the spec which should actually be in the install section

Comment (andreipope) [reply content/portworx-install-with-kubernetes/cloud/aws/aws-kops/plan.md]: @harsh-px Oh, it's already in the install section. I forgot to remove it from the planning page. It's fixed now. Thanks

Comment (andreipope) [reply content/portworx-install-with-kubernetes/cloud/aws/aws-kops/plan.md]: @harsh-px Changes are made. Please review this when you have some time.

**Comment by vilasd** [comment]
@andreipope resolve the conflicts and repost the PR

**Pull request closed without merging**